Add location data to proposals export and import

PierreMesure opened the pull request:
#### :tophat: What? Why?

WARNING: This PR is a draft. It is still lacking tests.

Lets Decidim export proposals with their location metadata (address, latitude, longitude) and reimport from a file.

#### :pushpin: Related Issues
- Related to [MetaDecidim#16871](https://meta.decidim.org/processes/roadmap/f/122/proposals/16871), #1223, #5186 
- Fixes #7062

#### Testing
- Export proposals from a Proposals component with geotagging
- Reimport them back in another component

#### :clipboard: Checklist

- [ ] :question: **CONSIDER** adding a unit test if your PR resolves an issue.
- [X] :heavy_check_mark: **DO** check open PR's to avoid duplicates.
- [X] :heavy_check_mark: **DO** keep pull requests small so they can be easily reviewed.
- [X] :heavy_check_mark: **DO** build locally before pushing.
- [ ] :heavy_check_mark: **DO** make sure tests pass.
- [X] :heavy_check_mark: **DO** make sure any new changes are documented in `docs/`.
- [X] :heavy_check_mark: **DO** add and modify seeds if necessary.
- [ ] :heavy_check_mark: **DO** add CHANGELOG upgrade notes if required.
- [ ] :heavy_check_mark: **DO** add to GraphQL API if there are new public fields.
- [X] :heavy_check_mark: **DO** add link to MetaDecidim if it's a new feature.
- [ ] :x:**AVOID** breaking the continuous integration build.
- [X] :x:**AVOID** making significant changes to the overall architecture.

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!

andreslucena commented:
> WARNING: This PR is a draft. It is still lacking tests.

Let me know when this is ready to be reviewed @PierreMesure

PierreMesure commented:
Hi @andreslucena ! I think it is ready although I wasn't able to write the additional tests I was initially planning for (I'm a terrible test engineer 🙈).

While having a look at the existing ones, I noticed that @lahdeero actually had the three fields I added in its example data. What did you have in mind @lahdeero? Did you want to add them to the import/export mechanism later on?

But that means that it shouldn't be too hard to adapt the specs to include the new behaviour I added.

lahdeero commented:
@PierreMesure I think you are referring to these lines: https://github.com/decidim/decidim/blob/develop/decidim-dev/lib/decidim/dev/test/rspec_support/component.rb#L310-L311 , these "imports.examples" are actually made by @ahukkanen . Maybe it should be something like this, that it would match example files: 
```ruby
translated.call("title") + %w(body) + translated.call("translatable_text") + %w(address coordinates/latitude coordinates/longitude),
locales.map { "Title text" } + ["Body text"] + locales.map { "Translatable text" } + ["Fake street 1", 1.0, 1.0]
```

andreslucena commented:
@lahdeero thanks for your quick response. I decided to drop the "coordinates/" prefix. 

This PR was superseded by #8679 - let's move the discussion there.
